core/src/com/cloud/agent/api/MigrateCommand.java

```diff
@@ -68,4 +93,79 @@ public String getHostGuid() {
     public boolean executeInSequence() {
         return executeInSequence;
     }
+
+    public static class MigrateDiskInfo {
+        public enum DiskType {
+            FILE, BLOCK;
+
+            @Override
+            public String toString() {
+                switch(this) {
```

Comment:
This does not make much sense. It is an enum, you will not be able to "instantiate" anything different from your constants. You can simply return this.name.toLowerCase, if you only want the name of the enum in lower case

Reply:
Thanks for catching this. I was initially returning something different than the simple lowercase version of the name of the enum, but then I went back to that simple approach and didn't change the code to reflect that simplicity.